parm/fv3lam_rrfs.xml

```diff
@@ -141,82 +141,90 @@
        <scale>3.0</scale>
        </param>
 
+       <param>
+       <shortname>CICE_ON_HYBRID_LVL</shortname>
```

Comment:
@AndrewBenjamin-NOAA this variable is not listed in post_avblflds.xml
but is listed in post_avblflds_raphrrr.xml

I know Wen coordinated with GSL to unify post_avblflds.xml so I would suggest asking GSL whether CICE is still a valid field.

Reply:
I will double check with GSL. I isolated the netcdf array that needs to be added to the diag table for the parallels (qqi) but it's possible it has not yet been added to the parallels. But either way, if it's not in post_avblflds.xml, it won't make it on the file. Who would be the best GSL contact to confirm this field?

Reply:
correct.

@AndrewBenjamin-NOAA could you remove this variable from your PR?